CMakeLists.txt

```diff
@@ -27,41 +27,45 @@ option(MRC_BUILD_DOCS "Enable building of API documentation" OFF)
 option(MRC_BUILD_LIBRARY "Whether the entire MRC library should be built. If set to OFF, only the pieces needed for a target will be built. Set to ON if installing the library" ON)
 option(MRC_BUILD_PYTHON "Enable building the python bindings for MRC" ON)
 option(MRC_BUILD_TESTS "Whether or not to build MRC tests" ON)
+option(MRC_ENABLE_CODECOV "Enable gcov code coverage" OFF)
+option(MRC_ENABLE_DEBUG_INFO "Enable printing debug information" OFF)
 option(MRC_USE_CCACHE "Enable caching compilation results with ccache" OFF)
 option(MRC_USE_CLANG_TIDY "Enable running clang-tidy as part of the build process" OFF)
 option(MRC_USE_CONDA "Enables finding dependencies via conda instead of vcpkg. Note: This will disable vcpkg. All dependencies must be installed first in the conda environment" ON)
 option(MRC_USE_IWYU "Enable running include-what-you-use as part of the build process" OFF)
-option(MRC_ENABLE_CODECOV "Enable gcov code coverage" OFF)
 
 set(MRC_RAPIDS_VERSION "22.08" CACHE STRING "Which version of RAPIDS to build for. Sets default versions for RAPIDS CMake and RMM.")
-
 set(MRC_CACHE_DIR "${CMAKE_SOURCE_DIR}/.cache" CACHE PATH "Directory to contain all CPM and CCache data")
 mark_as_advanced(MRC_CACHE_DIR)
 
+enable_testing()
+
 # CMake path
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 list(APPEND CMAKE_PREFIX_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
-include(cmake/setup_package_manager.cmake)
+# Load morpheus utils and update CMake paths
+set(MORPHEUS_UTILS_VERSION "23.01.01-alpha" CACHE STRING "Version of Morpheus utils")
+set(MORPHEUS_UTILS_RAPIDS_CMAKE_VERSION ${MRC_RAPIDS_VERSION})
+set(MORPHEUS_UTILS_RAPIDS_CPM_INIT_OVERRIDE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/rapids_cpm_package_overrides.json")
+include(load_morpheus_utils)
 
-enable_testing()
-
-# Add the RAPIDS cmake helper scripts
-include(cmake/import_rapids_cmake.cmake)
-
-# Default to using "NATIVE" for CUDA_ARCHITECTURES to build based on GPU in system
-if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
-  set(CMAKE_CUDA_ARCHITECTURES "NATIVE")
-  message(STATUS "CMAKE_CUDA_ARCHITECTURES was not defined. Defaulting to '${CMAKE_CUDA_ARCHITECTURES}' to build only for local architecture. Specify -DCMAKE_CUDA_ARCHITECTURES='ALL' to build for all archs.")
-endif()
+# Configure project package manager
+morpheus_utils_initialize_package_manager(
+    MRC_USE_CONDA
+    BUILD_SHARED_LIBS
+    MRC_VCPKG_TOOLCHAIN
+    MRC_VCPKG_DEFAULT_BINARY_CACHE
+    )
 
-rapids_cuda_init_architectures(mrc)
+# Configure CUDA architecture
+# NOTE: This MUST occur before any 'project' calls because of rapids_cmake requirements.
+morpheus_utils_initialize_cuda_arch(mrc)
 
 project(mrc
-  VERSION 22.11.00
-  LANGUAGES C CXX
-)
-
+    VERSION 22.11.00
+    LANGUAGES C CXX
+    )
```

Comment:
We need to sync on CMake formatters.

We recently added a config for the VS Code extension CMake Language Support to allow free-form indenting which was adding a lot of diffs.

Is this a CLion formatter? If so, we should get our VS Code extensions and your CLion config playing nice together.